packages/core-components/src/components/AutoLogoutProvider/index.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export * from './AutoLogoutProvider';
+export * from './ConfigBasedAutoLogoutProvider';
+
+/** @internal */
+export { LAST_SEEN_ONLINE_STORAGE_KEY } from './disconnectedUsers';
```

Comment:
```suggestion
```

I think this export be removed entirely right since we only use this key internally in the package for now? You should still be able to import it from within the package. It's easy to add it if we need access to the key from another package.